### PR TITLE
[cmake] Run fuzzing and shaping tests

### PR DESCRIPTION
**Pull request by ebraminio**
PTAL

### CODE REVIEW

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -1 +1,6 @@
 add_subdirectory(api)
+add_subdirectory(shaping)
+# Not ready yet
+if (NOT WIN32)
+  add_subdirectory(fuzzing)
```
Comment (khaledhosny) [inline test/CMakeLists.txt]: This should be checking for MSVC, the fuzzing is a clang/gcc feature.

Comment (ebraminio) [reply test/CMakeLists.txt]: Done.

**Comment by ebraminio** [comment]
Hi @khaledhosny, can you give me a hand on msys2 Windows issues? I guess hb-fuzzer executable is not on the expected place or something.

**Comment by ebraminio** [comment]
My guess is msys2 is using system's python instead msys2 one so it generates incorrect paths but don't know how to resolve if that is the case.

**Comment by ebraminio** [comment]
I guess I figured it out (by dropping `python` and letting env decide about it), lets see if that make it working.

Other than this, fuzzing and shaping test are disabled on MSVC CI tests as we don't have HB_BUILD_UTILS enabled on appveyor yet and I guess they have still path issues. I suggest fixing their issue sometime later.

**Comment by ebraminio** [comment]
> I guess I figured it out (by dropping python and letting env decide about it), lets see if that make it working.

Didn't work :(

**Pull request merged**